### PR TITLE
Handle rpmlint warning: no-changelogname-tag

### PR DESCRIPTION
**Pull request by bayucan**
#### Bug/feature Description
* pbspro.spec.in file is missing %changelog entry

#### Affected Platform(s)
* Linux/Unix

#### Cause / Analysis / Design / Solution
* Fix "no-changelogname-tag" warning in rpmlint by adding a %changelog entry.

#### Testing logs/output
* [rpmlint.bef.txt](https://github.com/PBSPro/pbspro/files/2671288/rpmlint.bef.txt)
* [rpmlint.aft.txt](https://github.com/PBSPro/pbspro/files/2671289/rpmlint.aft.txt)
* [rpmlint.diffs.txt](https://github.com/PBSPro/pbspro/files/2671290/rpmlint.diffs.txt)

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [X] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [X] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [X] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by bayucan** [comment]
@PBSPro/pbspro-maintainers : I've squashed and rebased. Feel free to move this forward as soon as the checks have finished running.

**Comment by mike0042** [comment]
Looks like https://github.com/PBSPro/pbspro/pull/921 is going to finish the CI checks first, so you'll need to rebase once it's merged.

File: pbspro.spec.in [hunk 1]
```diff
@@ -489,3 +489,56 @@ fi
...
+* Wed Jun 15 2016 Michael Karo <mike0042@gmail.com> - 1.4
+- Build PBS Pro under non-OHPC OBS instance
+* Fri Jun 3 2016 Nithin Johnson <nithin.johnson@altair.com> - 1.3
+- Add timezone and python-xml as dependencies
+* Wed May 25 2016 Michael Karo <mike0042@users.noreply.github.com> - 1.2
+- Update version to 14.0.1
+* Mon May 23 2016 arungrover <arun.grover@altair.com> - 1.1
+- Change to make sure that unsupported hook files are not compiled and packaged.
+* Thu May 12 2016 Hiren Vadalia <hiren.vadalia@altair.com> - 1.0
+- Initial commit of pbspro
```
Comment (mike0042) [inline pbspro.spec.in]: I just realized the corresponding changes to pbspro.spec are not part of this PR. Please run autogen.sh and include pbspro.spec with your changes.

Comment (bayucan) [reply pbspro.spec.in]: @mike0042 : Ok, I'll rebase once #921 completes.
On the subject of pbspro.spec, I thought it was sufficient just to make the change in pbspro.spec.in since autogen.sh will auto-generate pbspro.spec. Sure, I can include the corresponding pbspro.spec generated by autogen.sh. So does this mean this should also be the procedure to be followed by all developers when they make a change to pbspro.spec.in? Let me know as that will be the procedure I will communicate to the pbspro-maintainers when approving pbspro.spec.in changes...

Comment (mike0042) [reply pbspro.spec.in]: @bayucan : I'm trying to recall the reason we didn't simply remove pbspro.spec when we added pbspro.spec.in. It was originally added to build for OpenSUSE Tumbleweed. It looks like there are a couple Windows batch files that use it to obtain the PBS Pro version. I think these could be changed to reference the default version defined in configure.ac. There are also some scripts in .travis/ that reference it, but that happens after autogen.sh has been run. I don't have any objections to removing it. I think we really just wanted one place to store the default version, and that became configure.ac. We could remove it as part of this PR or file a ticket for it. Until then, both files should be updated.

**Comment by bayucan** [comment]
@mike0042 : For this PR, I'll just update both pbspro.spec.in and pbspro.spec files, but will file a ticket to recommend removing pbspro.spec altogether.

**Comment by bhroam** [comment]
@bayucan I believe pbspro.spec is checked in for a reason.  I can't recall the exact reason.  I think it has to do with something needing the version info.  Maybe postinstall on windows?  Windows doesn't call configure.

**Comment by mike0042** [comment]
@bhroam : Correct, Windows was the issue. However, the Windows batch scripts can get the same version identifier from configure.ac (currently 19.0.0). If you want to assign a version other than the default you do so when you call configure... ./configure PBS_VERSION=19.1.1
I'm not certain how this is done for Windows.

**Comment by bayucan** [comment]
Looks like another PR, #919 is going in first. I'll wait for that as well, and then I'll squash my commits and rebase.

**Comment by bayucan** [comment]
I've squashed and rebased.

**Comment by mike0042** [comment]
Please sign your commit @bayucan.

**Comment by bayucan** [comment]
Re-signed the commit.
